src/scss/_main.scss

```diff
@@ -864,32 +883,81 @@ nav {
   }
 }
 
-.logs .-focus {
-  background: var(--color-focus);
+.logs-table::-webkit-resizer {
+  background: var(--color-bg-dark);
+
+  border: 2px solid var(--color-bg-dark);
+  border-bottom: 3px solid var(--color-bg-light);
+  border-right: 3px solid var(--color-bg-light);
 }
 
-.line a {
-  text-decoration: none;
+.logs-table::-webkit-scrollbar {
+  width: 0.7rem;
+}
 
-  &:hover,
-  &:focus {
-    text-decoration: underline;
+.logs-table::-webkit-scrollbar-thumb {
+  background-color: var(--color-bg-light);
+  border-radius: 1rem;
+  border: 2px solid var(--color-bg-dark);
+}
+
+.logs-header {
+  padding: 0.25rem 0;
+  border-bottom: 1px solid var(--color-slate);
+  background: var(--color-bg-dark);
+  width: 100%;
+  z-index: 1;
+}
+
+.logs-header .actions {
+  padding: 0.3rem;
+
+  .buttons {
+    justify-content: flex-end;
+    margin-right: 0.3rem;
```

Comment:
so, i actually meant adding a modifier class to https://github.com/go-vela/ui/blob/master/src/scss/_buttons.scss#L226-L229 and building that out. however, this is clearer. we can deal with that in a different PR.